src/Context/index.tsx

```diff
@@ -159,14 +159,18 @@ function GameContextProvider(props: { children: ReactNode }) {
         }
     }, [dispatch]);
 
-    const startNewGame = useCallback(() => {
-        socket?.emit(USER_EVENTS.CREATE_GAME, (gameCode: string) => {
-            dispatch({
-                type: USER_EVENTS.CREATE_GAME,
-                payload: gameCode,
+    const startNewGame = useCallback((newGameId: string = '') => {
+        if (newGameId) {
```

Comment:
can we call these either both `id` or both `code`?

Reply:
sure, any preference? they are listed as both throughout the repo so i didn't know which to pick

Reply:
mmk, we'll worry about this after the other PRs are merged then